js/__tests__/gamestate-test.js

```diff
@@ -78,21 +78,62 @@ describe("GameState", function() {
     // play last card drawn
     state.selectCard(0, "hand", "bob")
     state.selectCard(0, "hand", "bob")
-
+   
     // go to next turn
     state.makeMove({op: "refreshCards"})
-
+   
     // get a card that implements direct damage
     state.draw({"name":"eve"}, {"damage":2, "cost":0})    
+   
     // play last card drawn
     state.selectCard(0, "hand", "eve")
     state.selectCard(0, "opponentBoard", "eve")
 
     // permanent leaves board and goes to trash
+    // bob
+    expect(state.current().board.length == 0).toEqual(true)
```

Comment:
Can you make this simpler by using toEqual directly:
expect(state.current().board.length).toEqual(0)